chsdi/models/vector/bafu.py

```diff
@@ -2326,3 +2326,36 @@ class LandesForstInventarVegetationsHoehenModellRelief(Base, LandesForstInventar
     __bodId__ = 'ch.bafu.landesforstinventar-vegetationshoehenmodell_relief'
 
 register(LandesForstInventarVegetationsHoehenModellRelief.__bodId__, LandesForstInventarVegetationsHoehenModellRelief)
+
+
+class LandesforstinventarRegion:
+    __table_args__ = ({'schema': 'wald', 'autoload': False})
+    __template__ = 'templates/landesforstinventar-region.mako'
```

Comment:
I am not sure if you can call the same template for different registries. It did not work when I tried it for one of my layers but could have also an other reason. But it could be the issue of your tool-tip

Reply:
I've seen it working in other cases ...

Reply:
OK then it was something else in my case (I did not spend much time debugging). 
I'll leave this template to you to debug, but you can let me know if I can be of any help.